lib/absinthe/pipeline.ex

```diff
@@ -138,6 +138,7 @@ defmodule Absinthe.Pipeline do
       {Phase.Schema.Validation.Result, pass: :initial},
       Phase.Schema.FieldImports,
       Phase.Schema.Validation.KnownDirectives,
+      Phase.Document.Validation.KnownArgumentNames,
```

Comment:
Probably shouldn't stick this here - should I duplicate this phase for `Schema`? Maybe trim it down to just directives?